sdk/eventhub/event-hubs/src/eventHubReceiver.ts

```diff
@@ -316,6 +337,9 @@ export class EventHubReceiver extends LinkEntity {
           );
           await this.detached(receiverError);
         } else {
+          if (this._aborter) {
+            this._aborter.removeEventListener("abort", this._onAbort);
+          }
```

Comment:
This is the case when receiver is already re-connecting. We shouldnt remove the listener in this case.